src/Service/MailService.php

```diff
@@ -120,6 +127,31 @@ public function sendOrderConfMail(Order $order, $orderStateId)
         );
     }
 
+    /**
+     * @param Order $order
+     * @param $orderStateId
+     * @throws \PrestaShopDatabaseException
+     * @throws \PrestaShopException
+     */
+    public function sendNewOrderMail(Order $order, $orderStateId)
+    {
+        if (!Module::isEnabled('ps_emailalerts')) {
+            return;
+        }
+        /** @var \Ps_EmailAlerts $emailAlertsModule */
+        $emailAlertsModule = Module::getInstanceByName('ps_emailalerts');
```

Comment:
are you sure you need to do it like that ? :o depent on another module ? If so atleast if this module is enabled first before doing any changes. Secondly maybe entirely it would be way better approach to copy that modules functionality and put it here ?